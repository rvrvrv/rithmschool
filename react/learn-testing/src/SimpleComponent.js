import React from 'react';

const SimpleComponent = ({ first, last, favFood }) => (
  <div>
    <h1>Hello there!</h1>
    <h2>Welcome!</h2>
    <p>First Name: {first}</p>
    <p>Last Name: {last}</p>
    <p>Favorite Food: {favFood}</p>
  </div>
);

export default SimpleComponent;
