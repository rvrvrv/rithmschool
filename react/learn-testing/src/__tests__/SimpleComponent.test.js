import React from 'react';
import SimpleComponent from '../SimpleComponent';
import renderer from 'react-test-renderer';

describe('<SimpleComponent />', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<SimpleComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
