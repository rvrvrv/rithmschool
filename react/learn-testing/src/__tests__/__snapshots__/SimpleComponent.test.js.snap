// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimpleComponent /> matches the snapshot 1`] = `
<div>
  <h1>
    Hello there!
  </h1>
  <h2>
    Welcome!
  </h2>
  <p>
    First Name: 
  </p>
  <p>
    Last Name: 
  </p>
  <p>
    Favorite Food: 
  </p>
</div>
`;
